src/test/java/group1/mips_simulator/FileReaderTest.java

```diff
@@ -13,7 +14,12 @@ class FileReaderTest {
     @Test
     void readBinaryFile() {
         FileReader underTest = new FileReader();
-        Vector<Instruction> result = underTest.readBinaryFile("src/test/java/group1/mips_simulator/testFile.txt");
```

Comment:
i actually dont know if this is a fix or if messed things up here while i was pulling/merging from origin
it just highlighted red and i did their suggested fix, we might want to ignore this branch (or even delete it?)